.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "third_party/spdlog"]
 	path = third_party/spdlog
 	url = https://github.com/gabime/spdlog.git
+[submodule "third_party/tabulate"]
+	path = third_party/tabulate
+	url = git@github.com:p-ranav/tabulate.git
```

Comment:
Can user without github id clone the submodule ?

Reply:
Hard to say, that depends on how we handle the cases of other submodules.